style.css

```diff
@@ -1,4 +1,74 @@
 body {
-background-color: #232323
+  color: #ffffff;
+  font-family: 'Ubuntu', sans-serif;
+  font-size: 15px;
+  width: auto;
```

Comment:
I don't think it's needed

Reply:
Do you mean font size, width or both? Please specify.

Reply:
Just `width: auto`